ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -516,4 +516,15 @@ CREATE TABLE participant_events_create_filter (
 
 CREATE INDEX idx_participant_events_create_filter_party_template_seq_id_idx ON participant_events_create_filter(party_id, template_id, event_sequential_id);
 CREATE INDEX idx_participant_events_create_filter_party_seq_id_idx ON participant_events_create_filter(party_id, event_sequential_id);
-CREATE INDEX idx_participant_events_create_seq_id_idx ON participant_events_create_filter(event_sequential_id);
\ No newline at end of file
+CREATE INDEX idx_participant_events_create_seq_id_idx ON participant_events_create_filter(event_sequential_id);
+
+CREATE TABLE transaction_metering (
+    application_id VARCHAR NOT NULL,
```

Comment:
Yes user-ids are a subset of application-ids. It might make sense to intern application_ids. I'm a bit wary of interning them without a story for garbage collecting unused ones.

How much extra effort do we accrue by not interning them now, but only doing it later once it has become the top priority?

Reply:
not much, application_id is everywhere already, this is probably the smallest part. good point with the garbage collection there, let's get back to this later :+1: